tests/test_practice_make_perfect.py

```diff
@@ -0,0 +1,222 @@
+import sys
+
+import pytest
+
+from codecademy.practice_make_perfect import is_int
+from codecademy.practice_make_perfect import factorial
+from codecademy.practice_make_perfect import reverse
+from codecademy.practice_make_perfect import scrabble_score
+from codecademy.practice_make_perfect import count
+from codecademy.practice_make_perfect import product
+from codecademy.practice_make_perfect import median
```

Comment:
You can chain import from the same package:
```python
from codecademy.practice_make_perfect import count, factorial, is_int, median, product, reverse, scrabble_score
```
Also please always sort your import alphabetically for better readability and easier code maintenance.
See PEP8: http://legacy.python.org/dev/peps/pep-0008/#imports